.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
+
+services:
```

Comment:
You can use `services: docker`, instead of starting a new line. The same applies to all other fields (`before_script`, `script`, etc.).